### PR TITLE
fix(base64): use arweave utils for browser friendly buffer conversion

### PR DESCRIPTION
**Pull request by atticusofsparta**

### CODE REVIEW

**Review by fedellen — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ar.io/sdk",
-  "version": "1.0.4",
+  "version": "1.0.3",
```
Comment (fedellen) [inline package.json:3]: should this be going to alpha?

File: src/utils/base64.ts [hunk 2]
```diff
@@ -14,14 +14,16 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
+import Arweave from 'arweave';
```
Comment (fedellen) [inline src/utils/base64.ts:17]: this import can always be tricky in a non web/node directory

Comment (atticusofsparta) [reply src/utils/base64.ts:17]: gotchya - still figuring out if this is the best way to do it, considering other options like https://www.npmjs.com/package/base64url at the moment.

**Comment by dtfiedler** [comment]
Replace with https://github.com/ar-io/ar-io-sdk/pull/73

**Pull request closed without merging**